docs/docs/Contributing/contributing-telemetry.mdx

```diff
@@ -13,18 +13,18 @@ To opt out of telemetry, set the `LANGFLOW_DO_NOT_TRACK` or `DO_NOT_TRACK` e
 
 ## Data that Langflow collects
 
-### Run {#2d427dca4f0148ae867997f6789e8bfb}
+### Run
```

Comment:
Avoid headings with nothing between them (see last bullet here: https://developers.google.com/style/headings#heading-and-title-format )